code/globalincs/vmallocator.h

```diff
@@ -1,6 +1,7 @@
 #ifndef _VMALLOCATOR_H_INCLUDED_
 #define _VMALLOCATOR_H_INCLUDED_
 
+#include <algorithm>
```

Comment:
Cause the header already used things from algorithm. It just never was an issue, since it was (before) always included somewhere where algorithm was already included prior. Since I added it into pstypes.h, where algorithm was not included, it would error otherwise